### PR TITLE
Update circleci-images to better display next-gen images.

### PR DESCRIPTION
**Pull request by teesloane**
Updates to our convenience image document. This change reorganizes the order of paragraphs to first display next-gen convenience images  over legacy documentation.

### CODE REVIEW

File: jekyll/_cci2/circleci-images.md [hunk 1]
```diff
@@ -415,6 +387,8 @@ that a given tag has the same meaning across images!
 {% endfor %}
 </ul>
 
+
+<br/>
```
Comment (teesloane) [inline jekyll/_cci2/circleci-images.md:391]: My local build needed a bit of space between the generated lists of images and the below `Note:` so I added a BR.

File: jekyll/_cci2/circleci-images.md [hunk 2]
```diff
@@ -358,9 +319,18 @@ The following packages are installed via `curl` or other means.
...
+   accepted.
+1. Old versions of software will not be rebuilt. Once an upstream image stops
+   building the tag for a specific release, say Node.js v8.1.0, then we stop
+   building it too. This means other tools in that image, such as `npm` in this
+   example, will no longer be updated either.
+1. We don't support building preview, beta, or release candidate images tags. On
+   occasion they'll be available but these tags tend to cause our build system
+   for Convenience Images to fail. If you need a non-stable release of a
+   language, we suggest installing it via [an orb](https://circleci.com/orbs/)
+   or a custom Docker image instead.
```
Comment (teesloane) [inline jekyll/_cci2/circleci-images.md:333]: I am wonder ing if the next-generation conveniences images have become commonplace enough to warrant removing this section?

File: jekyll/_cci2/circleci-images.md [hunk 3]
```diff
@@ -358,9 +320,18 @@ The following packages are installed via `curl` or other means.
 ## Out of scope
 {: #out-of-scope }
 
-1. If an image isn't listed above, it is not available. As the Convenience Image program is revamped, proposals for new images are not currently being accepted.
-1. Old versions of software will not be rebuilt. Once an upstream image stops building the tag for a specific release, say Node.js v8.1.0, then we stop building it too. This means other tools in that image, such as `npm` in this example, will no longer be updated either.
-1. We don't support building preview, beta, or release candidate images tags. On occasion they'll be available but these tags tend to cause our build system for Convenience Images to fail. If you need a non-stable release of a language, we suggest installing it via [an orb](https://circleci.com/orbs/) or a custom Docker image instead.
+1. If an image isn't listed above, it is not available. As the Convenience Image
```
Comment (thomasr8) [inline jekyll/_cci2/circleci-images.md:323]:
```suggestion
1. If an image is not listed above, it is not available. As the Convenience Image
```

File: jekyll/_cci2/circleci-images.md [hunk 4]
```diff
@@ -358,9 +320,18 @@ The following packages are installed via `curl` or other means.
...
+   program is revamped, proposals for new images are not currently being
+   accepted.
+1. Old versions of software will not be rebuilt. Once an upstream image stops
+   building the tag for a specific release, say Node.js v8.1.0, then we stop
+   building it too. This means other tools in that image, such as `npm` in this
+   example, will no longer be updated either.
+1. We don't support building preview, beta, or release candidate images tags. On
```
Comment (thomasr8) [inline jekyll/_cci2/circleci-images.md:330]:
```suggestion
1. We do not currently support building preview, beta, or release candidate images tags. On
```

Comment (thomasr8) [reply jekyll/_cci2/circleci-images.md:330]:
```suggestion
1. We do not support building preview, beta, or release candidate images tags. On
```

File: jekyll/_cci2/circleci-images.md [hunk 5]
```diff
@@ -358,9 +320,18 @@ The following packages are installed via `curl` or other means.
...
+   occasion they'll be available but these tags tend to cause our build system
```
Comment (thomasr8) [inline jekyll/_cci2/circleci-images.md:331]:
```suggestion
   occasion they will be available but these tags tend to cause our build system
```

**Review by thomasr8 — APPROVED** [review]
Some minor comments you may wish to incorporate, but overall, this looks really good!

**Pull request merged**